contracts/Vault1.sol

```diff
@@ -1,6 +1,25 @@
 //SPDX-License-Identifier: Unlicense
 pragma solidity ^0.8.4;
 
+import "@openzeppelin/contracts/utils/math/SafeMath.sol";
+
 contract Vault1 {
```

Comment:
@cal-a-mari - The vault does not support ERC20 tokens. The deposit and withdraw functions only support ether. You need to be able to deposit/withdraw ERC20 tokens.